lib/ansible/modules/cloud/ovirt/ovirt_permissions.py

```diff
@@ -70,6 +70,9 @@
     namespace:
         description:
             - Namespace of the authorization provider, where user/group resides.
+    quota_name:
+        description:
+            - Name of the quota which you want to work with.
```

Comment:
Name of the quota to assign permission. Works only with C(object_type) I(data_center).

Reply:
Please fix this and we can merge.